README.md

```diff
@@ -50,5 +50,145 @@ Pull requests are welcome for minor bug fixes or for a feature you are working o
 ---
 
 [0]: https://github.com/UdacityAndroidDevScholarship/blood-donation/issues
-[1]: https://console.firebase.google.com
 
+
+
+
+## Understanding MVP arch.
+
+The project has a package name `base`.
+It contains mainly 6 files
+
```

Comment:
@ritesh-singh could you validate these changes?

Reply:
I am not in favor.
We are spoon feeding others, as in telling them create this, create that, create a contract and all. Let them come up with their own after reading about MVP.
And since, we are not going to use the databinding library and changes he made to  base package, this readmefile doesn't makes sense now.